app/src/views/home/home_screen.tsx

```diff
@@ -93,7 +93,7 @@ export default function HomeScreen({
         {/* Current Challenge Description  */}
         <ChallengeDescription
           dBChallengeDescription={titleChallenge}
-          onTimerFinished={() => console.log("Timer Finished")}
+          onTimerFinished={() => console.info("Timer Finished")}
```

Comment:
What's the difference here ?

Reply:
Basically I put it as info as it is not an important comment and I can un-show it safely inside tests.

Reply:
Ok great!!